src/main/java/frc/robot/GameRobotContainer.java

```diff
@@ -92,31 +86,46 @@ public GameRobotContainer() {
   }
 
   private void configureBindings() {
-    /* Reset Robot */
+    // Reset Robot
     teleopEnabled.onTrue(retractIntake());
 
-    /* Driver Controller */
+    // Driver Controller
     driverController.start().onTrue(Commands.runOnce(drivetrain::seedFieldRelative));
-    driverController.a()
-        .onTrue(deployIntake())
-        .onFalse(retractIntake());
+    //driverController.a()
+       // .onTrue(deployIntake())
+       // .onFalse(retractIntake());
 
-    (driverController.b().and(m_IntakeSubsystem.isDeployed)).debounce(0.1).onTrue(retractIntake());
-    (driverController.b().and(m_IntakeSubsystem.isDeployed.negate())).debounce(0.1).onTrue(deployIntake());
+    // (driverController.b().and(m_IntakeSubsystem.isDeployed)).debounce(0.1).onTrue(retractIntake());
+    // (driverController.b().and(m_IntakeSubsystem.isDeployed.negate())).debounce(0.1).onTrue(deployIntake());
 
-    m_IntakeSubsystem.noteTriggered.onTrue(retractIntakePassToPB());
+    //m_IntakeSubsystem.noteTriggered.onTrue(retractIntakePassToPB());
 
-    /* Operator Controllers */
+    driverController.a().whileTrue(Commands.startEnd(
```

Comment:
You didn't push the changes we did in the beginning. The naming the commands part.